test-common/canton/it-lib/src/main/com/daml/CantonFixture.scala

```diff
@@ -119,7 +121,9 @@ trait CantonFixtureWithResource[A]
     else
       com.daml.fs.Utils.deleteRecursively(cantonTmpDir)
 
-  final protected def ports: Vector[Port] = suiteResource.value._1
+  final protected def ports: Vector[Port] = suiteResource.value._1.map(_._1)
+  // List of tuples of ledger api port then admin api port
```

Comment:
could we add something like `case class LedgerPorts(ledgerPort, apiPort)`?

Reply:
Yeah thats a good idea, I'll do that